specification/securityinsights/data-plane/Microsoft.SecurityInsights/preview/2024-02-01-preview/ThreatIntelligence.json

```diff
@@ -0,0 +1,192 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Sentinel Upload API",
+    "description": "Spec for Microsoft Sentinel Upload API.",
+    "version": "2024-02-01-preview"
+  },
+  "host": "api.ti.sentinel.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/workspaces/{workspaceId}/threatintelligencestixobjects:upload": {
+      "post": {
+        "x-ms-examples": {
+          "Upload Stix Objects": {
+            "$ref": "./examples/threatintelligence/UploadStixObjects.json"
+          }
+        },
+        "summary": "Upload Stix objects to the workspace",
+        "description": "Upload an array of Stix objects to the workspace specified",
+        "operationId": "ThreatIntelligenceStixObjects_Upload",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "workspaceId",
+            "description": "The workspace ID for the Stix objects to be uploaded.",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "maxLength": 36,
+            "pattern": "^[{]?[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}[}]?$"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Body of the request with an array of Stix objects and other properties",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/UploadStixObjectsRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The API returns 200 when at least one Stix object is successfully validated and published.",
+            "schema": {
+              "$ref": "#/definitions/UploadStixObjectsResponse"
+            }
+          },
+          "default": {
+            "description": "Upload Stix Objects API service error response.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "security": [
+          {
+            "azure_auth": [
+              "user_impersonation"
+            ]
+          }
+        ],
+        "x-ms-pageable": {
```

Comment:
Are you saying that you cannot leave off the `x-ms-pageable`?

Reply:
Hi @mpodwysocki , for the multiple endpoints, I just checked with my team. The special endpoint is only for 1 special customer. We dont expect others know about that endpoint. I think we can just leave it out of this swagger

Reply:
I removed the x-ms-pageable and I got the following issue:
"Operation might be pageable. Consider adding the x-ms-pageable extension."
![image](https://github.com/Azure/azure-rest-api-specs/assets/63322431/dd58c75b-cf00-46b9-95ff-ab72e77093a0)
 If I added "x-ms-pageable", the automation thinks this is a LIST api and asking me to do more things for LIST api